controllers/humiofilteralert_controller.go

```diff
@@ -218,6 +218,5 @@ func (r *HumioFilterAlertReconciler) logErrorAndReturn(err error, msg string) er
 }
 
 func sanitizeFilterAlert(filterAlert *humioapi.FilterAlert) {
-	filterAlert.ID = ""
```

Comment:
This was merged to master, so not sure if it is a merge of master into this branch/rebase gone wrong as it should not show up in this PR. You may need to do the same for aggregate alerts though, but removing this line and the same line for scheduled search is not supposed to be in this PR

Reply:
Not sure why it is showing up on this PR. Let me remove it.